setup.py

```diff
@@ -47,6 +47,7 @@ def get_packages(package):
     "asgiref>=3.3.4",
     "click>=7.*",
     "h11>=0.8",
+    "h2>=4.0.0",
```

Comment:
We want to do the same as httpx: https://github.com/encode/httpx/blob/ab64f7c41fc0fbe638dd586fecf0689c847109bb/setup.py#L66